pages/stakeholder-and-user-interviews.md

```diff
@@ -30,3 +30,7 @@ To build consensus about the problem statement and research objectives.
 ## Applied in government research
 
 No PRA implications. The PRA explicitly exempts direct observation and non-standardized conversation, 5 CFR 1320.3(h)3. See the methods for [Recruiting](../recruiting/) and [Privacy](../privacy/) for more tips on taking input from the public.
+
+## Additional resources
+
+ - [&ldquo;Build empathy with stakeholder interviews, part 1: Preparation&rdquo; by Andrew Maier](https://18f.gsa.gov/2016/06/20/build-empathy-with-stakeholder-interviews-part-1-preparation/)
```

Comment:
Actually, hang about. I think we've used quotation marks inconsistently, I'll double check and get back to you on that.